environment.yml

```diff
@@ -8,7 +8,7 @@ dependencies:
   - matplotlib=3.1.3
   - pandas=1.0.3
   - toml=0.9.4
-  - h5py=2.9.0
+  - h5py=2.10.0
```

Comment:
Is this needed or just keeping up with the latest versions?

Reply:
Yes, it [broke the build](https://travis-ci.com/github/ScottishCovidResponse/data_pipeline_api/builds/174195416#L507) because of a missing member of `h5py` - it wasn't really obvious to me from the docs that it wouldn't be available in 2.9.0 (in fact it _looks_ like it _is_ available), but there we go.